app/config/site-config.yml

```
@@ -3,7 +3,11 @@
   transmission_method: shared_email
   transmission_method_configuration:
     email: <%= ENV['NYC_HRA_EMAIL'] %>
+  sso:
+    provider: nyc
```

```suggestion
    provider: nyc_dss
```

I'm not sure whether this config is accessible from the devise initialization but it would be nice to see if we can get the various config in here (tenant ID, client ID, client secret), just to keep all the environment variables visible in one place. We could take that on later.